app/Repositories/Review/ReviewPhotoRepository.php

```diff
@@ -44,4 +44,12 @@ public function getByReview(int $reviewId): Collection
     {
         return ReviewPhoto::where('review_id', $reviewId)->get();
     }
+
+    public function getByPlace(int $placeId): Collection
+    {
+        return ReviewPhoto::select('review_photos.*')
+            ->leftJoin('reviews', 'reviews.id', '=', 'review_photos.review_id')
+            ->leftJoin('places', 'places.id', '=', 'reviews.place_id')
+            ->get();
```

Comment:
you didn't use a parameter $placeId here =)

Reply:
Fixed this) 
*facepalm*